src/main/java/Homework_number5.java

```diff
@@ -0,0 +1,119 @@
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Homework_number5 {
+
+    public static void main(String[] args) {
+
+        System.out.println(findSymbolOccurance("abbccc", 'c'));
+        System.out.println(findWordPosition("aaabbmmm", "bbb"));
+        System.out.println(reverseString("Hello"));
+        System.out.println(isPalindrome("abccba"));
+
+        playGame();
+
+    }
+
+    /* Створити метод findSymbolOccurance. Метод приймає як параметр рядок і символ.
+    Необхідно обчислити, скільки разів символ зустрічається в переданому рядку
+    і повернути це числове значення. */
+    public static int findSymbolOccurance(String inputString, char inputChar) {
+        int counter = 0;
+        for (int i = 0; i < inputString.length(); i++) {
+            if (inputString.charAt(i) == inputChar) {
+                counter++;
+            }
+        }
+        return counter;
+    }
+
+    /* Створити метод findWordPosition. Метод приймає як параметри
+    два рядки (source, target). Необхідно з'ясувати, чи є target (підрядок)
+    частиною рядка source. Якщо так, тоді повернути номер позиції (індекс)
+    першого елемента підрядка у рядку, інакше повернути -1. */
+    public static int findWordPosition(String source, String target) {
```

Comment:
Please add null validation for both parameters

Reply:
done